src/Compute/Compute/Strategies/ComputeRp/Images.json

```diff
@@ -0,0 +1,84 @@
+{
+    "Linux":  {
```

Comment:
NIT: one space after a colon.

Reply:
This is automatically generated from a PowerShell script, so I cannot change the format of the json.